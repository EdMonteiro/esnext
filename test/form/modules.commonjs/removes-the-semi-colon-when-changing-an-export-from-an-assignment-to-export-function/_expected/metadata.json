{
  "modules.commonjs": {
    "imports": [],
    "exports": [
      {
        "type": "named-export",
        "bindings": [
          {
            "exportName": "foo",
            "localName": "foo"
          }
        ],
        "node": {
          "type": "ExpressionStatement",
          "expression": {
            "type": "AssignmentExpression",
            "operator": "=",
            "left": {
              "type": "MemberExpression",
              "object": {
                "type": "Identifier",
                "name": "exports",
                "typeAnnotation": null
              },
              "property": {
                "type": "Identifier",
                "name": "foo",
                "typeAnnotation": null
              },
              "computed": false
            },
            "right": {
              "type": "FunctionExpression",
              "id": null,
              "params": [],
              "body": {
                "type": "BlockStatement",
                "directives": [],
                "body": []
              },
              "generator": false,
              "async": false,
              "returnType": null,
              "typeParameters": null
            }
          }
        }
      }
    ],
    "directives": []
  }
}