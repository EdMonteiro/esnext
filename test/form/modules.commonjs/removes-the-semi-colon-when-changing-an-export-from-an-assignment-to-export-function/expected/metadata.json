{
  "modules.commonjs": {
    "imports": [],
    "exports": [
      {
        "bindings": [
          {
            "exportName": "foo",
            "localName": "foo"
          }
        ],
        "node": {
          "type": "ExpressionStatement",
          "expression": {
            "type": "AssignmentExpression",
            "operator": "=",
            "left": {
              "type": "MemberExpression",
              "computed": false,
              "object": {
                "type": "Identifier",
                "name": "exports"
              },
              "property": {
                "type": "Identifier",
                "name": "foo"
              }
            },
            "right": {
              "type": "FunctionExpression",
              "body": {
                "type": "BlockStatement",
                "body": []
              },
              "expression": false,
              "generator": false,
              "id": null,
              "params": []
            }
          }
        }
      }
    ],
    "directives": []
  }
}